README.md

```diff
@@ -165,6 +160,9 @@ history.
 Contributions to SeineSailor are welcome! If you have any suggestions, bug reports, or feature requests, please open an
 issue on the [GitHub repository](https://github.com/SeineAI/SeineSailor).
 
+Important! Please read [Contributing_README.md](docs/Contributing_README.md) to understand how we separate production 
```

Comment:
@SeineSailor Can you comment on whether this line is necessary, or any improvement suggestions?